package.json

```diff
@@ -32,7 +31,7 @@
   "scripts": {
     "build": "npm run prod && npm run dev",
     "prod": "webpack --config webpack.config.babel.js -p",
-    "dev": "webpack --config webpack.config.babel.js -d --devtool hidden --output-file=tota11y.js",
+    "dev": "webpack --config webpack.config.babel.js -d --devtool hidden --output=tota11y.js",
```

Comment:
Looks like `tota11y.js` is no longer being generated because of this line. Is that possible?